### ISSUE TITLE
Abbreviate name differently?

### ISSUE THREAD

**Issue by mvz** [mawww/kakoune#259]
The name of this editor is shortened to 'kak' in the command and the script file extensions. Unfortunately, 'kak' means 'poo' in Dutch, so perhaps a different three-letter combination can be used (e.g., kkn?).

**Comment by mawww** [comment]
Hello,

Yeah, I heard about that, it is unfortunate, however I am not sure it warrant a name change, there is so many languages out there and its hard to find a 3 letter, pronounceable, word that wont be an insult or a bad word in one of them.

Even 'vi' is an old french word for penis...

I heard it meant poo in Afrikaans, do you mean its more generally all Dutch derivated languages that use 'kak' for 'poo' ?

**Comment by mvz** [comment]
> do you mean its more generally all Dutch derivated languages that use 'kak' for 'poo'?

Yes. It's not the most commonly used word, but all Dutch and Flemish speakers would know it.

I would normally agree that changing the name because it's problematic in one language goes a bit too far, but I think it's particularly unfortunate in this case since Dutch is the mother tongue of Vim's creator.

An unpronounceable abbreviation might fare better in this respect. Compare the commands `rm`, `cd`, and the file extension `rb`.

**Comment by dumblob** [comment]
Well, to me it's rather funny than serious issue. In Czech (which is my mother tongue) `kak` is the result of the action `kakat`, which is a rather polite way of saying `to shit` (actually, `kakoune` is used literally in Czech as imperative meaning `you small child` and it's a common addressing of children up to cca 4 years age).

In Russian, `kak` means `as` or `how`, but it's written using the Russian alphabet, not latin (https://translate.google.com/#en/ru/as%0Ahow).

In general, I wouldn't be concerned about this that much. It's still way better than other IT terms - e.g. Czechs use the abbreviation `API` as `apíčko`, which is a diminutive of `api` (which has no meaning in Czech - it's just the lower-case variant of the English abbreviation `API`), but unfortunately to many non-technical people it sounds inpolite, because `píča` means `cunt` (`... a pičko ...` would mean imperative form of `... and you smaller cunt ...`).

In other words, there are many IT terms and slang variations which are not easily acceptable by the public, so `kak` is not so bad :wink:.

But of course, 2-character name would make it faster to type and also contribute to the kak's goal *Faster as in less keystrokes* :wink:
